src/Shell/CommandListShell.php

```diff
@@ -145,7 +146,7 @@ public function getOptionParser()
             'help' => 'Get the listing as XML.',
             'boolean' => true
         ])->addOption('version', [
-            'help' => 'Prints the currently installed version of CakePHP.',
+            'help' => 'Prints the currently installed version of CakePHP. (deprecated - use `cake --version` instead)',
```

Comment:
I do not understand. If you want to see the help, you have to use `cake --version` instead? That does not make much sense to me.